buildpipeline/pipelinejobs.groovy

```diff
@@ -13,36 +13,46 @@ def branch = GithubBranchName
-	['Debug', 'Release'].each { configurationGroup ->
-		['Linux x64'].each { osName ->
-            // Runs the portable-linux.groovy pipeline on the target Helix queues mentioned in the pipeline.  Currently:
-            // CentOS 7.3, RedHat 7.3, Debian 8.7, Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10, openSuSE 42.2 and Fedora 25
-
+    ['Debug', 'Release'].each { configurationGroup ->
+        ['Linux x64'].each { osName ->
             // One for just innerloop.
-            linuxPipeline.triggerPipelineOnEveryGithubPR("Portable ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':false])
+            linuxPipeline.triggerPipelineOnEveryGithubPR("${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':false])
             // Add one for outerloop
-            linuxPipeline.triggerPipelineOnGithubPRComment("Portable Outerloop ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':true])
-		}
-	}
+            linuxPipeline.triggerPipelineOnGithubPRComment("Outerloop ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':true])
+        }
+    }
 }
 
-// Create a pipeline for portable windows
-def windowsPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/portable-windows.groovy')
+def osxPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/osx.groovy')
 ['netcoreapp'].each { targetGroup ->
-	['Debug', 'Release'].each { configurationGroup ->
-		['Windows x64'].each { osName ->
-            // Runs the portable-windows.groovy pipeline on the target Helix queues mentioned in the pipeline.  Currently:
-            // Windows 10, Windows 7, Windows 8.1 and Windows Nano
-
-            // One for just innerloop
-            windowsPipeline.triggerPipelineOnEveryGithubPR("Portable ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':false])
+    ['Debug', 'Release'].each { configurationGroup ->
+        ['OSX x64'].each { osName ->
+            // One for just innerloop.
+            //TODO: Add manual trigger for OSX for now until we ramp up machines. Turn this to triggerPipelineOnEveryGithubPR when we remove OSX from netci.groovy.
+            osxPipeline.triggerPipelineOnGithubPRComment("${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':false])
             // Add one for outerloop
-            windowsPipeline.triggerPipelineOnGithubPRComment("Portable Outerloop ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':true])
-		}
-	}
+            osxPipeline.triggerPipelineOnGithubPRComment("Outerloop ${osName} ${configurationGroup} Build", ['Config':configurationGroup, 'OuterLoop':true])
+        }
+    }
+}
+
+def windowsNameMap = ['netcoreapp':'Windows x64',
+                      'netfx':'NetFx',
+                      'uap':'UWP',
+                      'uapaot':'UWP .NET Native',
```

Comment:
The TestILC package isn't available but you can do a product and IL test build but you cannot do native test build.